apps/webbsite/src/libs/notion/getNotionClient.ts

```diff
@@ -0,0 +1,26 @@
+import { NotionAPI } from 'notion-client';
+
+let notionClient: NotionAPI;
+
+/**
+ * Lazy load the notion client
+ * @returns the notion client
+ */
+export default function getNotionClient(): NotionAPI {
+  if (notionClient) {
+    return notionClient;
+  }
+
+  const NOTION_TOKEN_V2 = process.env.NOTION_TOKEN_V2;
+  const NOTION_ACTIVE_USER = process.env.NOTION_ACTIVE_USER;
```

Comment:
A potential solution here is described in this post:
https://stackoverflow.com/a/73115430/8610613

If we go that route, we will want to have a condition to check if its a CI environment or local development so we can still develop locally fine. Not sure how this will work with netlify but worth exploring?